CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `@tailwind/vite` now generates unique CSS bundle hashes when the Tailwind-generated CSS changes ([#13218](https://github.com/tailwindlabs/tailwindcss/pull/13218))
 - Inherit letter spacing in form controls ([#13328](https://github.com/tailwindlabs/tailwindcss/pull/13328))
 - Ensure `build` command is executed when using `--output` instead of `-o` ([#13369](https://github.com/tailwindlabs/tailwindcss/pull/13369))
+- Workaround for `@tailwind/vite` hanging in dev server mode ([#13380](https://github.com/tailwindlabs/tailwindcss/pull/13380))
```

Comment:
I would try to phrase this as an action starting with a verb, something like:

> Prevent `@tailwindcss/vite` from hanging in serve mode

Package is `@tailwindcss/vite` too, not `@tailwind/vite` 👍 

Should also make the same change to the earlier changelog entry:

```diff
- `@tailwind/vite` now generates unique CSS bundle hashes when the Tailwind-generated CSS changes
- Generate unique CSS bundle hashes in `@tailwindcss/vite` when Tailwind-generated CSS changes
```